task0/LinearRegression_beyzaNur.ipynb

```diff
@@ -0,0 +1,1586 @@
+{
```

Comment:
<p>Are you sure that you included the right file? I cant see the solutiıns here. It seems that you adjusted the code above, but did you reported it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bogaziciai/assignment/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
I updated the file with the answers, I think I had a problem with saving first time maybe.

Reply:
If an issue is solved here, we click "Resolve conversation" to make this PR pretty. Feel free to do that.